tests/Oryx.RuntimeImage.Tests/Ruby/RubyImagesTest.cs

```diff
@@ -18,7 +18,7 @@ public RubyImagesTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
```

Comment:
I'm curious to why we've changed to `SkippableFact`?

Reply:
ah, yes, so I am somewhat unfamiliar with xunit and so this may not have been the correct way to approach the problem but changed this because `Skip.If` was throwing an exception locally that flat out failed the test. This change allowed the test to instead skip instead of failing when executed locally (I believe that it should still execute unchanged on the ci build agent)
![image](https://user-images.githubusercontent.com/107068277/174163644-d1e7c920-5a74-4f07-9725-a55b517b3f77.png)

Reply:
Got it!